_hoh_081_enemies/araragi.md

```diff
@@ -46,6 +46,10 @@ notes:
       - 'Time for roughly 3 auto-attacks before it restarts (another Goobbue''s
       Grief)'
 job_specifics:
+  DRK:
+    difficulty: 'Easy if you can LoS Goobbue''s Grief well'
+    notes:
+      - 'Use TBN at the end of Goobbue rotation'
```

Comment:
I'm guessing you mean during the third Goobue's Grief, so it blocks the auto-attacks? Can you clarify?